src/test/unit/test_tracer.py

```diff
@@ -183,28 +181,26 @@ def lambda_test_function(event, context):
     assert Configuration.should_report == "123"
 
 
-def test_wrapping_with_print_override(context):
-    @lumigo_tracer(enhance_print=True)
+def test_wrapping_print_happy_flow(context):
+    @lumigo_tracer()
     def lambda_test_function(event, context):
-        print("hello\nworld")
+        print("hello")
         return 1
 
     with CaptureOutput() as capturer:
         assert lambda_test_function({}, context) == 1
-        assert Configuration.enhanced_print is True
-        assert "RequestId: 1234 hello" in capturer.get_lines()
-        assert "RequestId: 1234 world" in capturer.get_lines()
+        assert any(line == "hello" for line in capturer.get_lines())
 
 
-def test_wrapping_without_print_override(context):
-    @lumigo_tracer()
+def test_wrapping_enhanced_print_backward_compatible(context):
+    @lumigo_tracer(enhance_print=True)
     def lambda_test_function(event, context):
         print("hello")
+        assert Configuration.enhanced_print is False
```

Comment:
I think that `Configuration` shouldn't have the `enhanced_print` property anymore, isn't it?

Reply:
I agree. Deleted :)